### DISCUSSION TITLE
Analysis of 26a and 26b

### DISCUSSION THREAD

**Discussion by JulioTBacmeister** [NCAR/amwg_dev#181] [TF5: Coupled SE/MOM6]
L58 runs with better tuned clouds

**Comment by JulioTBacmeister** [comment]
26a #178 and 26b #179 have clouds retuned via CLUBB parameter clubb_c8 to obtain RESTOM closer to 0.  Inital look after nearly 30 years of SY is here:
![image](https://user-images.githubusercontent.com/9221710/189702284-76272191-9ce4-4056-bf2e-46217df69083.png)

**Reply by cecilehannay** [reply]
ADF diags are available:
- 26a: [ADF diags ](https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.026a/atm/)
- 26b: [ADF diags ](https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.026b/atm/)

**Comment by JulioTBacmeister** [comment]
26a -39 years, 26b - 41 years.  Lab Sea Ice fraction in 26b increased dramatically between year 35 and year 41.
![image](https://user-images.githubusercontent.com/9221710/190455249-94001606-89d3-4811-a018-15548ebaafb4.png)

**Reply by adamrher** [reply]
Is the lab sea state in 026a acceptable? 

If no, than since 26b jumps to the lab sea state of 26a around year 35, does that indicate we maybe we went to hard on the cloud tunings? My understanding is that 26b starts from a warm and salty ocean state in the lab-sea, so if it still leads to problematic ice cover, than I think it means we overshot the mark. 

But I don't want to get ahead of myself; the first question seems like the top priority.

**Comment by JulioTBacmeister** [comment]
Updated time series: 026a==> Year 48; 026b==> Year 54

![image](https://user-images.githubusercontent.com/9221710/190916560-93b9f1cd-6b21-409e-ae4f-5650ba11865a.png)

**Comment by megandevlan** [comment]
Updated cloud forcing/surface fields as well: 
<img width="1149" alt="Screen Shot 2022-09-19 at 9 26 44 AM" src="https://user-images.githubusercontent.com/42625059/191054525-6df2997f-3356-43cd-a051-9135d60f1352.png">

<img width="980" alt="Screen Shot 2022-09-19 at 9 27 14 AM" src="https://user-images.githubusercontent.com/42625059/191054634-0254f7ed-5d9b-42b4-94c3-efa04f00444d.png">

Based on Julio's global TS time series, I'm wondering if 026a/b are still gradually cooling. Similar to Adam, I'm also not sure if this is an acceptable ice cover fraction for the Lab Sea; it looks like the region is a bit too cold and fairly ice covered in March during years 35-45.

**Comment by cecilehannay** [comment]
In Julio’s diag (icefrac over lab sea), 26a and 26b look no worse than 16 (I.e borderline but acceptable).  

We need the assessment from the ocean people because we know the icefrac doesn’t give the full picture.

**Reply by cecilehannay** [reply]
This is really weird. I see that GitHub resent old posts last night. It was comments that I previously posted. These comments were resent last night as a new comment.  Does someone has an explanation for this? Could I be hacked ? (Jesse?).
